crates/edr_provider/src/data.rs

```diff
@@ -2774,6 +2774,16 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn fork_metadata_fork_mode() -> anyhow::Result<()> {
+        let fixture = ProviderTestFixture::new_forked(None)?;
+
+        let fork_metadata = fixture.provider_data.fork_metadata().unwrap();
```

Comment:
Nice to have: change the `unwrap` into an `expect("some explanation")` to be explicit about why this is valid logic